packages/firebase/useRTDB/index.ts

```diff
@@ -19,9 +24,10 @@ export function useRTDB<T = any>(
 
   docRef.on('value', update)
 
-  tryOnUnmounted(() => {
-    docRef.off('value', update)
-  })
-
+  if (options.autoDispose) {
```

Comment:
Will need to do the object destucturing like `useFirebase` does to assign the default value.

Reply:
done with updates.